ethicml/data/tabular_data/synthetic.py

```diff
@@ -1,39 +1,57 @@
 """Class to describe features of the Synthetic dataset."""
 from warnings import warn
 
+from teext import PositiveInt
 from typing_extensions import Literal
 
 from ..dataset import Dataset
 
 __all__ = ["Synthetic", "synthetic"]
 
 
-def Synthetic() -> Dataset:  # pylint: disable=invalid-name
+def Synthetic(  # pylint: disable=invalid-name
+    scenario: Literal[1, 2, 3] = 1,  # pylint: disable=bad-whitespace
+    target: Literal[1, 2, 3] = 1,  # pylint: disable=bad-whitespace
+    fair: bool = False,
+    num_samples: PositiveInt = 1_000,
```

Comment:
does this line pass mypy? I'd have thought you'd need to do this:
```suggestion
    num_samples: PositiveInt = PositiveInt(1_000),
```